pyproject.toml

```diff
@@ -10,7 +10,6 @@ dependencies = [
     "httpx[http2] >=0.23.0",
     "opentelemetry-api >=1.20.0",
     "opentelemetry-sdk >=1.20.0",
```

Comment:
I don't know toml very well, but shouldn't the coma on the last array element be removed as well?

Reply:
As long as its the last element in the list, the trailing comma is optional. Having it does not break.